CMakeLists.txt

```diff
@@ -74,7 +81,7 @@ if (NOT BLT_LOADED)
   endif ()
 
   include(${BLT_SOURCE_DIR}/SetupBLT.cmake)
-endif()
+endif ()
```

Comment:
I don't think these space changes are necessary, and I think most of the CMakeLists are formatted with spacing before the `()`

Reply:
I was trying to make it consistent. If you look throughout this file there are endifs both with and without the space. Let me know if you don't want that in this PR and I'll take them out.

Reply:
No worries. It might have been better to do in a separate PR, but I thought I'd just put it here for simplicity :)